.github/workflows/autoupdate-preview.yaml

```diff
@@ -56,9 +60,24 @@ jobs:
             ## Using early preview changes
 
             ```sh
-            go get go.mongodb.org/atlas-sdk/v20240805005@dev-latest   
+            go get github.com/mongodb/atlas-sdk-go@preview-latest
```

Comment:
Any reason we need to change branch? 
Since this was merged, please update all wikis/documentation that depend on that branch name.

Reply:
i did it to make it more explicit that it's for Preview, nothing else depends on this branch name except the generated PR description, that was adapted in this PR

Reply:
sorry I meant there are not occurrences for string "dev-latest".
It's not that we renamed the branch name, but that we've started to use a new branch name for SDK Preview from now on